swagger.yaml

```diff
@@ -661,18 +661,18 @@ paths:
           schema:
             type: string
       requestBody:
-        description: Email of the user whose attribute is being updated.
+        description: Attribute update payload.
         required: true
         content:
           application/json:
             schema:
               type: object
               required:
-                - email
+                - value
```

Comment:
What is this value? I thought the point of this refactor was to make the endpoint more specific so rather than the user providing some generic/ flexible value e.g. SeveraOptIn string it only dealys with severa opt in?
Do we no longer need to pass the email here?

Reply:
Ok so as you have the severa opt in put below, can't this endpoint now just be removed?